src/main/java/pl/fracz/mcr/source/Line.java

```diff
@@ -0,0 +1,117 @@
+package pl.fracz.mcr.source;
+
+/*
+ 2013-10-23, fracz, first implementation
+ 2013-10-30, fracz, added syntax highlighting
+ 2014-02-26, fracz, added ability to add voice comment
+ */
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.text.Html;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import pl.fracz.mcr.comment.AbstractComment;
+import pl.fracz.mcr.comment.Comment;
+import pl.fracz.mcr.comment.CommentNotAddedException;
+
+import java.io.File;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * View that represents one line of code.
+ */
+@SuppressLint("ViewConstructor")
+public class Line extends LinearLayout implements Serializable {
+    private static final long serialVersionUID = 3076583280108678995L;
+    private static final int TWO = 2;
+
+    private final int _lineNumber;
+
+    private final String _lineOfCode;
+
+    // holds the line number
+    private TextView lineNumberView;
+
+    private TextView lineContent;
+
+    private SourceFile sourceFile;
+
+    private List<Comment> comments;
+
+    public Line(Context context, SourceFile sourceFile, int lineNumber,
+                String lineOfCode, boolean syntaxColor) {
+        super(context);
+        this.sourceFile = sourceFile;
+        this._lineNumber = lineNumber;
+        this._lineOfCode = lineOfCode;
+        setOrientation(LinearLayout.HORIZONTAL);
+
+        lineNumberView = new TextView(getContext());
+        lineNumberView.setText(String.format("%d.", lineNumber););
+        lineNumberView.setSingleLine();
+        lineNumberView.setWidth(30);
+        addView(lineNumberView);
```

Comment:
lineNumberView operations can be isolated into a function, maybe `setLineNumberView`?